### ISSUE TITLE
Code colorisé pour une meilleur compréhension

### ISSUE THREAD

**Issue by Skrol29** [jeantil/admission_post_bac#41]
Voici un lien vers la version colorisée du code avec 
- arguments de la fonction
- variables internes
- tables,
- commentaires perso

Moi, ça m'a permis de bien comprendre le code.

https://docs.google.com/document/d/1fZE7mh81Pb6MKJmw73BTKNxhunZT0QUn57GkX6g_7wQ/edit

**Comment by khena** [comment]
Je ne sais pas si tu es sur le drive officiel, j'avais commencé à faire les deux premières requêtes avec les italiques, les gras les soulignés selon les paramètres d'E/S
Tu n'aurais pas envie de faire un petit markdown par hasard, pour le wiki ça serait bien non ?

**Comment by jferard** [comment]
Effectivement, @Skrol29 et @khena, tout ça est bien utile pour différencier paramètres, variables locales et champs des tables et aurait sa place soit dans le wiki, soit dans un fichier à part (`abp-original.md`), mais comment cumuler la coloration syntaxique automatique du SQL et une mise en forme "à la main" ?
